src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -125,6 +128,22 @@ module BABYLON {
                 if (this._attached) {
                     this.getEngine().enableVR();
                 }
+                if(this._useStandingMatrix){
+                    if (!this._vrDevice.stageParameters.sittingToStandingTransform) {
+                        this._useStandingMatrixCallback(false);
+                    }else{
+                        this._standingMatrix = new Matrix();
+                        Matrix.FromFloat32ArrayToRefScaled(this._vrDevice.stageParameters.sittingToStandingTransform, 0, 1, this._standingMatrix);
+                        if (!this.getScene().useRightHandedSystem) {
+                            [2, 6, 8, 9, 14].forEach((num) => {
+                                if (this._standingMatrix) {
+                                    this._standingMatrix.m[num] *= -1;
+                                }
+                            });
+                        }
+                        this._useStandingMatrixCallback(true)

```

Comment:
Not sure I see the added value in calling the callback at all, but I understand this is needed since the callback needs to be called somewhere :)

Reply:
The callback allows the user to tell if the vr device they are using supports the standing matrix. It currently only works when using Vive/Oculus in firefox or chrome. I agree, adding a class variable to store the callback is somewhat odd. I think I want to keep the callback in the useStandingMatrix function scope.